internal/api/api.proto

```diff
@@ -16,6 +16,9 @@ message InstallResponse {}
 message PingRequest {}
 message PingResponse {}
 
+message StateRequest {}
+message StateResponse {string state = 1;}
```

Comment:
Do we really need state as a string in the code? It seems like `enum` should be enough: https://protobuf.dev/programming-guides/enum/